### PR TITLE
Add capture screenshots script

### PR DESCRIPTION
**Pull request by davidrapson**
Uses headless Chrome through [puppeteer](https://github.com/GoogleChrome/puppeteer) to grab screenshots of common pages at key breakpoints. Limits it to home and 10k pages for now.

~Next step is to push these to an s3 bucket somewhere.~

### CODE REVIEW

**Comment by davidrapson** [comment]
@mattandrews Going to come back to this, managed to get it reliably capturing one page but bails out as soon as I try additional pages, probably missing something obvious but too flaky so far. Cleaned this up to just add the script and not attempt to automate it for now, plan to run it manually every so often and upload to s3 for the moment.

File: bin/scripts/capture-screenshots [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    const sizeSlug = size ? `${size.width}x${size.height}` : '';
+    const parts = [urlSlug, sizeSlug, dateSlug].filter(_ => _).join('-');
+    return `${parts}.${FILE_FORMAT}`;
+}
+
+del.sync([`${DEST_PATH}/*.${FILE_FORMAT}`]);
+puppeteer.launch({ dumpio: true, args: ['--no-sandbox', '--disable-setuid-sandbox'] }).then(browser => {
+    function capturePageSizes(urlPath, size) {
+        return browser.newPage().then(page => {
+            return page.goto(`https://test.blf.digital${urlPath}`, { waitUntil: 'networkidle' }).then(() => {
```
Comment (mattandrews) [inline bin/scripts/capture-screenshots]: Can you read this from a config file? Don't want this endpoint exposed anywhere public really...

File: bin/scripts/capture-screenshots [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                        assert(fs.existsSync(screenshotFullPath));
+                        console.log(`${screenshotFullPath} saved 🎉 `);
+                        return page.close();
+                    });
+                });
+            });
+        });
+    }
+
+    const promises = [].concat(
```
Comment (mattandrews) [inline bin/scripts/capture-screenshots]: This may be me not understanding this syntax but why do this over just listing a regular array?

**Comment by mattandrews** [comment]
👍

**Pull request merged**